index.md

```diff
@@ -1,144 +1,51 @@
-
-#### Archived Projects
-
-- ![area](https://img.shields.io/badge/Geospatial-blue) &nbsp; [Granular Mapping](https://nhsx.github.io/nhsx-internship-projects/granular-mapping/)
-- ![technique](https://img.shields.io/badge/ImageSegmentation-lightgrey)
-![datamodality](https://img.shields.io/badge/Images-ff69b4) &nbsp;  [NHS GreenSpace](https://nhsx.github.io/nhsx-internship-projects/nhs-greenspace/)
-- ![area](https://img.shields.io/badge/NLP-yellow)
-![technique](https://img.shields.io/badge/SematicSearch-lightgrey)
-![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet) &nbsp; [NHS Semantic Search](https://nhsx.github.io/nhsx-internship-projects/nhs-semantic-search/)
-
----
-More information on the internship scheme can be found on the [NHS England internship scheme for innovation and analytics in health](https://www.nhsx.nhs.uk/key-tools-and-info/nhsx-analytics-unit/nhsx-internship-scheme-innovation-and-analytics-health/) website.
+# {{page.title}}
+
+These internships are aimed at current PhD students looking for an industrial placement of around 5-months with the right to work in the UK.  The projects are focussed on innovation in particular around getting the most value out of NHS data.  They often have a focus on emerging data science techniques and so we advertise mainly to data science programmes however previous interns have included disciplines such as clinical, mathematics, computer science and bioinformatics which have added huge value through the range of approaches and knowledge.
+
+For details around **how to apply** see the "Scheme Overview" page.  For details around **open projects** see the "Projects" page.
+### Current Projects 
+Currently our interns are working on the following six projects in two waves.  These are the original briefs they applied to.  Their work and outputs will be available on our organisation github.
+
+|:---:|--|
+|January 2023|
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/CogStack-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TextData-9cf) &emsp; &emsp; &emsp; &emsp; &emsp; |[Enriching Neurology Patient Information using MedCAT](https://nhsx.github.io/nhsx-internship-projects/enriching-neurology-information-medcat/)|
+|![area](https://img.shields.io/badge/Synthetic-brightgreen) &nbsp; ![technique](https://img.shields.io/badge/VAE-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[SynthVAE - Next Steps](https://nhsx.github.io/nhsx-internship-projects/synthvae-extension/)|
+|![area](https://img.shields.io/badge/Graphs-yellowgreen) &nbsp; ![technique](https://img.shields.io/badge/Hypergraphs-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Transforming Healthcare Data with Graph-based Techniques - Next Steps](https://nhsx.github.io/nhsx-internship-projects/transforming-healthcare-data-graph-based-sail-update/)|
+|June 2023|
```

Comment:
Add Wave number to standardise across pages

Reply:
Done